pod-operation/src/state_machine.rs

```diff
@@ -30,6 +31,8 @@ pub struct StateMachine {
 	state_transitions: EnumMap<State, Option<StateTransition>>,
 	io: SocketIo,
 	brakes: Brakes,
+	upstream_pressure_transducer: PressureTransducer,
+	downstream_pressure_transducer: PressureTransducer,
```

Comment:
Thought/nitpick, cc @samderanova: to have a common prefix, naming the variants by suffix might be more appropriate?

Reply:
Hmm I personally don't think it'll make very much difference. As long as these two fields are grouped together, I feel that's specific/readable enough.